bootstrapers/hychart/utils/fetchTranslations.ts

```diff
@@ -0,0 +1,20 @@
+export const fetchTranslations = async (
+  lang: string
+): Promise<Record<string, string | Record<string, string>>> => {
+  let translations: Record<string, string | Record<string, string>> = {};
+
+  try {
+    const baseUrl = "http://localhost:3000";
+    const url = `${baseUrl}/assets/locales/${lang}.json`;
+
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error("Network response was not ok");
+    }
+    translations = await response.json();
+  } catch (error) {
+    console.error("Error fetching translations:", error);
+  }
+
+  return translations;
+};
```

Comment:
I think this will not work in production, because we are running the project on port 80
In the article, they are loading the files directly from the path without the need to use Http request to fetch the files, I think that would be better and faster

Just please make sure that those JSONs are there in the final build (npm run build), and that when you do (npm run start) it's working like expected